docs/guides/zookeeper/scaling/vertical-scaling/vertical-scaling.md

```
@@ -136,15 +136,15 @@ spec:
 
 Here,
 
-- `spec.databaseRef.name` specifies that we are performing vertical scaling operation on `vscale` database.
+- `spec.databaseRef.name` specifies that we are performing vertical scaling operation on `zk-quickstart` database.
 - `spec.type` specifies that we are performing `VerticalScaling` on our database.
 - `spec.VerticalScaling.node` specifies the desired resources after scaling.
 - Have a look [here](/docs/guides/zookeeper/concepts/opsrequest.md#spectimeout) on the respective sections to understand the `timeout` & `apply` fields.
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)